src/jdk.compiler/share/classes/com/sun/tools/javac/comp/InferenceContext.java

```diff
@@ -551,8 +651,18 @@ List<Type> solveBasic(List<Type> varsToSolve, EnumSet<InferenceStep> steps) {
 
     @Override
     public String toString() {
-        return "Inference vars: " + inferencevars + '\n' +
-               "Undet vars: " + undetvars;
+        String result = "Inference vars: " + inferencevars + '\n' +
+               "Undet vars: " + undetvars + '\n';
+        /*
+        // keep it for debugging
```

Comment:
Should we control this with an `options.isSet` switch instead, maybe passing the options through Infer?

Reply:
yes we could do that, but then we will probably want to add a test for that option, etc, probably not worthy, also we could just drop this code